src/features/Send/useCases/ListSelectedTokens/ListSelectedTokensScreen.tsx

```diff
@@ -0,0 +1,176 @@
+import {useNavigation} from '@react-navigation/native'
+import * as React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {StyleSheet, TouchableOpacity, View, ViewProps} from 'react-native'
+import {FlatList} from 'react-native-gesture-handler'
+import {useQuery, UseQueryOptions} from 'react-query'
+
+import {Boundary, Button, Spacer} from '../../../../components'
+import {AssetItem} from '../../../../components/AssetItem'
+import globalMessages from '../../../../i18n/global-messages'
+import {TxHistoryRouteNavigation} from '../../../../navigation'
+import {useSelectedWallet} from '../../../../SelectedWallet'
+import {COLORS} from '../../../../theme'
+import {sortTokenInfos} from '../../../../utils'
+import {YoroiWallet} from '../../../../yoroi-wallets/cardano/types'
+import {useTokenInfo, useTokenInfos} from '../../../../yoroi-wallets/hooks'
+import {TokenId, TokenInfo, YoroiAmount, YoroiEntry, YoroiUnsignedTx} from '../../../../yoroi-wallets/types'
+import {Amounts} from '../../../../yoroi-wallets/utils'
+import {useSend} from '../../common/SendContext'
+import {AddTokenButton} from './AddToken/AddToken'
+import {DeleteToken} from './DeleteToken'
+import {useWhenEmptyAddToken} from './WhenEmptyAddToken'
+
+export const ListSelectedTokensScreen = () => {
+  const navigateTo = useNavigateTo()
+  const strings = useStrings()
+
+  const {targets, selectedTargetIndex, tokenSelectedChanged, amountRemoved, yoroiUnsignedTxChanged} = useSend()
+  const {amounts} = targets[selectedTargetIndex].entry
+
+  const selectedTokensCounter = Object.keys(amounts).length
+  useWhenEmptyAddToken()
```

Comment:
what is an empty token?

Reply:
can we prevent the user from adding an empty token?

Reply:
yap changed the logic in the Apply button, so this use case is no longer valid